src/model.ts

```diff
@@ -148,36 +154,48 @@ export class JSORMBase {
   static currentClass: typeof JSORMBase = JSORMBase
   static beforeFetch: BeforeFilter | undefined
   static afterFetch: AfterFilter | undefined
-  static jwtStorage: string | false = "jwt"
 
   private static _typeRegistry: JsonapiTypeRegistry
   private static _IDMap: IDMap
   private static _middlewareStack: MiddlewareStack
-  private static _credentialStorageBackend?: StorageBackend
-  private static _credentialStorage?: CredentialStorage
+  private static _credentialStorageBackend: StorageBackend
+  private static _credentialStorage: CredentialStorage
+  private static _jwtStorage: string | false = "jwt"
 
   static get credentialStorage(): CredentialStorage {
-    if (!this._credentialStorage) {
-      if (!this._credentialStorageBackend) {
-        if (this.jwtStorage && typeof localStorage !== "undefined") {
-          this._credentialStorageBackend = localStorage
-        } else {
-          this._credentialStorageBackend = new NullStorageBackend()
-        }
-      }
+    return this._credentialStorage
+  }
 
-      this._credentialStorage = new CredentialStorage(
-        this.jwtStorage,
-        this._credentialStorageBackend
-      )
+  static set jwtStorage(val : string | false) {
+    if (val !== this._jwtStorage) {
```

Comment:
What's the downside if this check were removed?

Reply:
using an in-memory store and running `ApplicationRecord.jwtStorage = ApplicationRecord.jwtStorage` would wipe out the record.  Otherwise nothing I can think of.